README.md

```diff
@@ -2,35 +2,69 @@
 
 > Twitter strategy for Überauth.
 
-### Setup
+## Installation
 
-Include the provider in your configuration for Überauth:
+1. Setup your application at [Twitter Developers](https://dev.twitter.com/).
 
-```elixir
-config :ueberauth, Ueberauth,
-  providers: [
-    twitter: [ { Ueberauth.Strategy.Twitter, [] } ]
-  ]
-```
+1. Add `:ueberauth_twitter` to your list of dependencies in `mix.exs`:
 
-Then configure your provider:
+    ```elixir
+    def deps do
+      [{:ueberauth_twitter, "~> 0.1"}]
```

Comment:
Because you're using a `git` dependency for `erlang-oauth`, users will need to specify it in their `deps` as `hex` doesn't manage this. So we should add that here.